packages/block-editor/src/store/selectors.js

```diff
@@ -1435,7 +1435,7 @@ export function getTemplateLock( state, rootClientId ) {
 
 	const blockListSettings = getBlockListSettings( state, rootClientId );
 	if ( ! blockListSettings ) {
-		return null;
+		return undefined;
```

Comment:
Here I'm making the assumption that `null` and `undefined` are the same thing. Maybe @jorgefilipecosta can confirm. 

The `updateBlockListSettings` was being called twice for each container block because of this.

Reply:
Yes, it should be the same thing.